### ISSUE TITLE
search packages can't work. only after switching to themes and return to packages

### ISSUE THREAD

**Issue by foxmuldercp** [atom/settings-view#553]
Use case:
 Open Packages -> Settings view -> Install packages, then enter some text in search field, but nothing doing.
 for get result I must change Packages button to Themes and switch again for a get search result.
 No 'Enter' key action here too

**Comment by benogle** [comment]
Closing in favor of #559
